text-dream/python/dream/README.md

```diff
@@ -0,0 +1,135 @@
+# DeepDream With BERT
+
+This repository contains functions to conduct deepdream experiments with BERT.
+All of the commands assume that they are executed from within the folder
+`python`.
+
+## Requirements
+
+- torch
+- pytorch_pretrained_bert
+- absl_py
+
+You can install them all by simply issuing: `pip3 install -r
+dream/requirements.txt`
+
+
+## Dream
+
+The `dream` binary can be used to do classic deepdream experiments with BERT.
+To be able to backpropagate right to the input tokens, we use a method with a
+smooth one-hot encoded input vector.
+This vector is fed through a softmax layer.
+To force this vector to get close to real tokens, we use temperature annealing.
+
+Example: `python dream/dream.py -output_dir ~/Downloads
```

Comment:
some places use python, some python3. unify. does this only support py3 or can both be used? document.